src/js/components/TextInput/__tests__/__snapshots__/TextInput-test.js.snap

```diff
@@ -436,6 +436,7 @@ exports[`TextInput close suggestion drop 4`] = `
       data-testid="test-input"
       id="item"
       name="item"
+      tabindex="-1"
```

Comment:
I think Travis might have trouble with this.

Reply:
I wondered about it! Thank you I will take it out!